ai_edge_torch/generative/test/test_model_conversion.py

```diff
@@ -27,22 +24,24 @@
 import numpy as np
 import torch
 
+from tensorflow.python.platform import googletest
 
-class TestModelConversion(unittest.TestCase):
+
+class TestModelConversion(googletest.TestCase):
   """Unit tests that check for model conversion and correctness."""
 
   def test_toy_model_with_kv_cache(self):
     config = toy_model_with_kv_cache.get_model_config()
-    pytorch_model = toy_model_with_kv_cache.ToyModelWithKV(config)
+    pytorch_model = toy_model_with_kv_cache.ToyModelWithKV(config).eval()
```

Comment:
could you explain a bit why we need `.eval()`?

Reply:
This sets the module in evaluation (inference) mode.
https://pytorch.org/docs/stable/generated/torch.nn.Module.html#torch.nn.Module.eval

`ai_edge_torch.convert` expects the module to be in this mode and emits a warning otherwise:
https://github.com/google-ai-edge/ai-edge-torch/tree/main/docs/pytorch_converter#providing-a-wrapper

It changes certain layers in the module preparing for inference. For example, for BatchNorm2d, it freezes the running statistics.